store/src/sqlite.rs

```diff
@@ -461,6 +497,24 @@ where
     V: Serializable + Send + Sync + Clone + 'static,
     MKT: crate::MetadataLocalKey + 'static,
 {
+    async fn load_all_data(&self) -> Result<Vec<V>, StoreError> {
```

Comment:
Why not referring a single function for load_all_data?

Reply:
rust won't allow that across different implementation of the store type (or it does using generics but I'm not there yet :))